pkg/odo/cli/url/create.go

```diff
@@ -61,15 +65,19 @@ func (o *URLCreateOptions) Complete(name string, cmd *cobra.Command, args []stri
 	} else {
 		o.urlName = args[0]
 	}
+	o.localConfigInfo, err = config.NewLocalConfigInfo(o.componentContext)
+
 	return
 }
 
 // Validate validates the UrlCreateOptions based on completed values
 func (o *URLCreateOptions) Validate() (err error) {
 
-	exists, err := url.Exists(o.Client, o.urlName, o.Component(), o.Application)
-	if exists {
-		return fmt.Errorf("The url %s already exists in the application: %s\n%v", o.urlName, o.Application, err)
+	// Check if exist
+	for _, localUrl := range o.localConfigInfo.GetUrl() {
```

Comment:
Just thinking of a name that indicates this returns a collection and not single url